DuckDuckGo/AutofillLoginDetailsViewController.swift

```diff
@@ -52,15 +52,19 @@ class AutofillLoginDetailsViewController: UIViewController {
         return barButtonItem
     }()
 
-    init(authenticator: AutofillLoginListAuthenticator, account: SecureVaultModels.WebsiteAccount? = nil, authenticationNotRequired: Bool = false) {
-        self.viewModel = AutofillLoginDetailsViewModel(account: account)
+    init(authenticator: AutofillLoginListAuthenticator, account: SecureVaultModels.WebsiteAccount? = nil, tld: TLD, authenticationNotRequired: Bool = false) {
+        self.viewModel = AutofillLoginDetailsViewModel(account: account, tld: tld)
         self.authenticator = authenticator
         self.authenticationNotRequired = authenticationNotRequired
         super.init(nibName: nil, bundle: nil)
         self.viewModel.delegate = self
 
     }
-    
+
+    deinit {
+        NotificationCenter.default.removeObserver(self)
```

Comment:
I think we don't need to explicitly call removeObserver on deinit anymore, so would be inclined to remove (although perhaps should put it to the team)